_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Using Tettra to Keep our Content Organized"
+date: 2020-09-17 11:00:00
+reviewed: 2020-09-17 11:00:00
+categories: ["team", "remote"]
+author: "abizzinotto"
+---
+
+At OmbuLabs we use [Tettra](https://tettra.com/) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](https://www.ombulabs.com/blog/team/agile/remote/organizing-team-knowledge-bases.html), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
+
+<!--more-->
+
+## Getting Started
+
+To make sure every team member knows how to navigate and use Tettra, we have a Using Tettra category. Under this category, we have created content to help team members get started with the tool, the integrations available and to get familiar with the structure of the knowledge base and how to navigate it.
+
+Here we have a style guide to explain how we write new pages, the documented structure of the knowledge base so every team member knows what each category is comprised of and where to find documents or add new content. This also includes our standard procedures to add new content or suggest new categories.
+
+## Knowledge Base Structure
+
+To keep our content organized, we use Tettra's categories and folders structure. Each category represents an area and each folder a division within that area. For example, under the Sales category, we have the Pre-Sales and Marketing folders.
+
+We also use Folders to keep our Draft pages before they are officially published to the whole team.
+
+<img src="/blog/assets/images/tettra/tettra_categories.jpg" alt="tettra categories" class="medium-img">
+
+## Adding Content
+
+Tettra allows you to create pages inside the tool as well as add a link to external documents (Google Doc or Google Sheet). This lets us create new pages as needed as well as link to our documents available through Google Docs and Google Sheets, keeping all the information in a single place.
+
+To make sure we keep things organized and standardized, we created a style guide for our Tettra pages that can be found under the Using Tettra category. Additionally, we also have a standard procedure for adding content.
```

Comment:
This is a little bit repetitive as you already mentioned the style guide in the Getting Started section. 😄 

Not a big deal though.

Reply:
This was on purpose :) I wanted to emphasize the style guide as a very common problem in knowledge bases is content being added in all sorts of different ways making it hard to read and update. Perhaps not the best way to go about it though...